searx/engines/odysee.py

```diff
@@ -0,0 +1,92 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# lint: pylint
+"""Odysee_ is a decentralised video hosting platform.
+
+.. _Odysee: https://github.com/OdyseeTeam/odysee-frontend
+"""
+
+import time
+from datetime import datetime
+
+# Engine metadata
+about = {
+    "website": "https://odysee.com/",
+    "wikidata_id": "Q102046570",
+    "official_api_documentation": None,
+    "use_official_api": False,
+    "require_api_key": False,
+    "results": "JSON",
+}
+
+# Engine configuration
+paging = True
+results_per_page = 20
+categories = ['images']
```

Comment:
```suggestion
categories = ['videos']
```

Reply:
Argh .. I haven't seen your review comments before I merge this PR / sorry .. I will fix it in a follow up.

Reply:
@Bnyro thanks a lot for your review comments / I would like to apologize again for overlooking this in my haste / I added your review in commit https://github.com/searxng/searxng/commit/4f8895c6de2659f9ca069dde7be8eb3714ed1494 on the master branch.

Reply:
No worries, thanks for doing the suggested cleanups!